iarbackend/settings/base.py

```diff
@@ -191,3 +191,5 @@
         },
     },
 }
+
+IAR_USERS_LOOKUP_GROUP = 'uis-iar-users'
```

Comment:
Can we also add something like this to assets.defaultsettings and add a system check to make it a required setting?
```python
"""
Required. Name of lookup group which user must be a member of to have modify access to IAR.

"""
IAR_USERS_LOOKUP_GROUP = None
```